bin/i18n/sync-codeorg-in.rb

```diff
@@ -67,18 +66,100 @@ def localize_dsl_content
     Script.all.each do |script|
       next unless ScriptConstants.i18n? script.name
```

Comment:
Should we also check if the script is "stable" here?

Reply:
@bethanyaconnor What do you mean by "stable" in this context?

Reply:
I believe that means the script is out of beta. There's a function `is_stable?` and I believe Course A 2018 is "stable" but Course A 2019 is not, for example.

Reply:
definitely!

Reply:
Oh hmm actually it seems like some hoc scripts aren't marked "stable"